src/tests/operate.test.js

```diff
@@ -0,0 +1,50 @@
+import operate from '../logic/operate';
+
+const objArr = [
+  {
+    numberOne: '10',
+    numberTwo: '5',
+    operation: '÷',
+  },
+  {
+    numberOne: '5',
+    numberTwo: '6',
+    operation: 'x',
+  },
+  {
+    numberOne: '26',
+    numberTwo: '0',
+    operation: '+',
+  },
+  {
+    numberOne: '8',
+    numberTwo: '4',
+    operation: '-',
+  },
+  {
+    numberOne: '14',
+    numberTwo: '7',
+    operation: '%',
+  },
+];
+
+const resultArr = ['2', '30', '26', '4', '0'];
+
+describe('calculator displays the correct result', () => {
+  test('operators return correct result', () => {
+    objArr.forEach((obj, i) => {
+      const result = operate(obj.numberOne, obj.numberTwo, obj.operation);
+      expect(result).toEqual(resultArr[i]);
+    });
+  });
+  test('division by 0 returns a message', () => {
+    const result = operate('5', '0', '÷');
+    expect(result).toBe("Can't divide by 0.");
+  });
+
+  test('any other operator throws an error message', () => {
+    expect(() => {
+      operate('5', '0', '/');
+    }).toThrow("Unknown operation '/'");
+  });
+});
```

Comment:
### Let us add the following tests.

- [ ] Should give the sum of two given numbers is the operator is +
- [ ] Should give the difference of two given numbers is the operator is -
- [ ] Should give the product of two given numbers is the operator is x
- [ ] Should give the percentage of the first number is the operator is %
- [ ] Should give the rest of the division is the operator is ÷
- [ ] Should throw an error if given a wrong number

Reply:
Hello Emmyn5600, the result of each operator is in resultArr